### PR TITLE
Moved Support to .org (#3360)

### PR DESCRIPTION
**Pull request by MichaelRupertDev**
docs: moved support options to www.borgbackup.org
(cherry picked from commit 71fbac94fbd0b15d3e302c1ed6d967331037bd4e)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -127,7 +127,7 @@ Spread the word, give feedback, help with documentation, testing or development.
 
 You can also give monetary support to the project, see there for details:
 
-https://borgbackup.readthedocs.io/en/stable/support.html#bounties-and-fundraisers
+https://www.borgbackup.org/support/support.html#bounties-and-fundraisers
```
Comment (ThomasWaldmann) [inline README.rst]: hmm, wasn't it support/free.html (vs. commercial.html) in the end?

Comment (MichaelRupertDev) [reply README.rst]: that it was. Gonna close this and re-do when fixes are merged.

Comment (ThomasWaldmann) [reply README.rst]: hmm, this is still todo, right?

Comment (MichaelRupertDev) [reply README.rst]: Nope, complete with my other backport PR

**Pull request closed without merging**